tornadox/solvers/ek1.py

```diff
@@ -374,14 +374,11 @@ def body_fun(s):
 
         # Condense both backward models into one.
         # (Technically speaking, this implements 'fixed-point smoothing')
-        # todo: if we do this somewhere else, perform_step_fn() can be reused
+        # todo: if we do this condensing somewhere else, perform_step_fn() can be reused
```

Comment:
What does this comment mean actually?

Reply:
if the line: 
linear-model \circ linear-model = new-linear-model
is performed not inside perform_step_fn, but outside of that, then perform_step_fn can be reused as a default ode-solver-step, instead of being tied to the saveat/checkpoint version

Reply:
I see, thanks